### ISSUE TITLE
PointsNodeMaterial opacityNode has no effect

### ISSUE THREAD

**Issue by wmcmurray** [mrdoob/three.js#23054]
**Describe the bug**

Defining an `opacityNode` on a `PointsNodeMaterial` has no effect (the resulting fragment shader remain unchanged).

**Live example**

[codepen example](https://codepen.io/wmcmurray/pen/bGoRoBV?editors=0010) (line 46 👀)

**Platform:**

 - Device: probably all
 - OS: probably all
 - Browser: probably all
 - Three.js version: r135

**Comment by wmcmurray** [comment]
**More details...**

I believe [this is the cause of the problem](https://github.com/mrdoob/three.js/blob/25c7f7a1c9349a652cc8fb0d7645593991756bf9/examples/jsm/renderers/webgl/nodes/WebGLNodeBuilder.js#L412-L416) : 

```js
this.addCodeAfterInclude(
  'fragment',
  'alphamap_fragment',
  `${opacityNode.code}\n\tdiffuseColor.a = ${opacityNode.result};`
);
```

because there is no `alphamap_fragment` chunk included in the fragment shader of `PointsMaterial` (so the snippet never get injected)

Replacing that part with this could "fix" the problem:

```js
this.addCodeAfterSnippet(
    'fragment',
    'vec4 diffuseColor = vec4( diffuse, opacity );',
    `${opacityNode.code}\n\tdiffuseColor.a = ${opacityNode.result};`
);
```

...but I suspect it will break other materials (I haven't checked).

I don't know what is the best way to fix this so... cc @sunag  👋🏼😄

**Comment by Mugen87** [comment]
@wmcmurray Do you mind verifying if #23064 (specifically the change to `WebGLNodeBuilder`) fixed this issue?

**Comment by Mugen87** [comment]
Should be fixed via #23064.

Updated live example: https://jsfiddle.net/tepb7osm/
